Replacement of the chip component of filters inside contingency list

flomillot opened the pull request:
## PR Summary

- Removed unused `color` imports and `equipmentColorsMap` for improved cleanup and code clarity.
- Updated `ContingencyListFilterBasedForm` to use the `ChipComponent` and added custom `chipProps` for better reusability and maintainability.
- Added additional translation mappings for `equipmentSearch` keys to enhance localization capabilities.

flomillot committed:
- Removed unused color imports and `equipmentColorsMap` for cleanup.
- Updated `DirectoryItemsInput` in `ContingencyListFilterBasedForm` to utilize `ChipComponent` and custom `chipProps`.
- Integrated additional translation mappings for `equipmentSearch` keys.

Signed-off-by: Florent MILLOT <florent.millot_externe@rte-france.com>

basseche commented on src/components/dialogs/contingency-list/filter-based/contingency-list-filter-based-form.tsx:
```diff
@@ -7,12 +7,13 @@
     CONTINGENCY_LIST_EQUIPMENTS,
+    ContingencyListEquipment,
     DescriptionField,
+    DirectoryItemChipWithHelperText,
```
it's OverflowableChipWithHelperText

flomillot committed:
Replaced `DirectoryItemChipWithHelperText` with `OverflowableChipWithHelperText` in `ContingencyListFilterBasedForm` for improved chip component handling.

Signed-off-by: Florent MILLOT <florent.millot_externe@rte-france.com>

flomillot committed:
Add equipmentShort translations to app wrappers

Included `equipmentShortEn` and `equipmentShortFr` in both English and French translations for app wrappers in `gridstudy-app` and `gridexplore-app`. This ensures consistency and support for the new `equipmentShort` localization across the applications.

Signed-off-by: Florent MILLOT <florent.millot_externe@rte-france.com>

flomillot committed:
Upgrade `@gridsuite/commons-ui` to version 0.137.0.

Signed-off-by: Florent MILLOT <florent.millot_externe@rte-france.com>

flomillot committed:
Remove unused `equipmentSearch` translations from app wrappers

Cleaned up redundant `equipmentSearchEn` and `equipmentSearchFr` entries from both English and French translations to streamline localization in app wrappers.

Signed-off-by: Florent MILLOT <florent.millot_externe@rte-france.com>
